.github/workflows/release.yml

```diff
@@ -0,0 +1,488 @@
+name: "Release Builds"
+on:
+  push:
+    branches:
+      - v[0-9].[0-9]**
+    tags:
+      - v[0-9].[0-9]**
+      - v[0-9].[0-9]**-**
```

Comment:
Would it be possible to manually run this on some arbitrary commit?

Reply:
Do you mean from a security standpoint, or a convenience standpoint?

In terms of convenience, I can set up a `workflow_dispatch` condition and have it provide a field you can enter to select a specific commit. It would then check out that commit and build all the binaries as usual.

In terms of security, there's a blurb at the top of the `Secrets` page:

>Secrets are environment variables that are encrypted. Anyone with collaborator access to this repository can use these secrets for Actions.

>Secrets are not passed to workflows that are triggered by a pull request from a fork. [Learn more](https://docs.github.com/actions/automating-your-workflow-with-github-actions/creating-and-using-encrypted-secrets).

So as long as the list of _collaborators_ is kept managed appropriately, using the `Secrets` in workflows is relatively safe; people can't just fork Taisei and run an arbitrary workflow to get the secrets.

Reply:
From a convenience standpoint; I've read up on security already.

> In terms of convenience, I can set up a workflow_dispatch condition and have it provide a field you can enter to select a specific commit. It would then check out that commit and build all the binaries as usual.

Sounds good to me. Maybe later we can also upgrade this to provide e.g. nightly autobuilds.